### ISSUE TITLE
GitHub issues from servo/servo not showing?

### ISSUE THREAD

**Issue by caugner** [jdm/bugsahoy#182]
When I visit https://www.joshmatthews.net/bugsahoy/?servo=1, I don't see any bugs:

![image](https://user-images.githubusercontent.com/495429/41204085-5b887c54-6ce0-11e8-8766-d2c584e2196d.png)

However, I would expect to see [these bugs](https://github.com/servo/servo/issues?q=is%3Aopen+is%3Aissue+label%3AE-easy):

![image](https://user-images.githubusercontent.com/495429/41204090-663089c6-6ce0-11e8-9fb9-f384045c64a1.png)

Is my expectation wrong or the GitHub integration broken?

**Comment by jdm** [comment]
The github integration is broken. Thanks for noticing!

**Comment by jdm** [comment]
My devtools console shows me:
```
XML Parsing Error: syntax error
Location: https://www.joshmatthews.net/bugsahoy/cgi-bin/githubapi.cgi?user=servo&name=servo&labels=E-easy
Line Number 1, Column 1:
```

**Comment by caugner** [comment]
Ah, the problem seems to be the shebang in [githubapi.cgi](https://github.com/jdm/bugsahoy/blob/master/cgi-bin/githubapi.cgi#L1) that doesn't specify the python version (either `python2` or `python3`, since there is no `python` symlink set on the system bugsahoy is running), see the output from https://www.joshmatthews.net/bugsahoy/cgi-bin/githubapi.cgi?user=servo&name=servo&labels=E-easy:

```
The following python binaries are available:

python2: Python 2.7.15
python3: Python 3.6.4

Please use the binary for the version of Python you want.
```

**Comment by caugner** [comment]
Probably a symlink `python -> python2` on the system would be the best solution, no?

**Comment by jdm** [comment]
Indeed! Good catch; fixed in b1e44a36f9ad35ab76b7b06d0359e237611f153c.
